scripts/source-archive

```diff
@@ -186,6 +186,7 @@ if __name__ == "__main__":
         vcsdate_unix, vcsdate))
     cmakeversion = tarfile.TarInfo("%s/cmake/GitVersion.cmake" % (prefix))
     cmakeversion.size = cmakeversionbuf.len
+    cmakeversion.mtime = time.time()
```

Comment:
Can you add a comment in the script itself explaining the reason why the `mtime` attribute of `TarInfo` need to be overriden?